test/val/val_id_test.cpp

```diff
@@ -1037,23 +1037,6 @@ TEST_F(ValidateIdWithMessage,
                         "component count does not match Result Type <id> '4's "
                         "vector component count."));
 }
-TEST_F(ValidateIdWithMessage, OpConstantCompositeMatrixColumnTypeBad) {
```

Comment:
This test gets caught by type checks now.

Reply:
Should we convert it to check for the type error instead of removing it?

Reply:
We do have tests separately for that so I figure it was redundant to add again. I can if you think there's value.